readme.md

```diff
@@ -74,9 +78,7 @@ intention is to create a virtuous cycle, examples:
 
 ## Community
 
-<!-- Join us on [matrix/irc][matrix-channel]! -->
-
-Currently, this repository is our primary community hub; however, we'll introduce something more interactive as things grow. At this time our community is small but
+Currently, this repository and our [matrix/irc][nim-works-matrix] are our primary community hubs; however, we'll introduce more as things grow. At this time our community is small but

```

Comment:
the _however,_ is satisfied by the semi-colon

Reply:
```suggestion
Currently, this repository and our [matrix/irc][nim-works-matrix] are our primary community hubs; we'll introduce more as things grow. At this time our community is small but
```